mtools/test/test_mlaunch_commandlines.py

```diff
@@ -117,15 +117,15 @@ def test_single(self):
         """mlaunch should start 1 node."""
         self.run_tool('init --single')
         cmdlist = [
-            set(['mongod', '--dbpath', '--logpath', '--port', '--fork'])
+            set(['"mongod"', '--dbpath', '--logpath', '--port', '--fork'])
```

Comment:
Standard command line options are repeated in most of the tests. Consider refactoring into reusable declarations (but not essential for this PR).